docs/en-us/1.1-USER_MANUAL.md

```diff
@@ -17,8 +17,8 @@
     <!--For the re-engraved SS event, you can enter `SSReopen-XX` to clear XX-1 ~ XX-9 levels at one time. Such as `SSReopen-IC`.-->
 
 - Fight options include `Use Sanity Potion`, `Use Originium`, `Perform Battles` and `Material`, you can specific any of them. The fight tasks stops once one of the specifications is met.
-  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Note that the potion is usually used one at a time which is the one default by the game client when current sanity is not enough for the next fight. However, Santity Samplers offer a smaller amount of sanity and thus might be used more than one at a time. Pay attention to this when specifying the number of potions to use.
-  - `Use Originium` specifies the number of Originium to use at most. It is used one at a time.
+  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Multiple medicines may be used at one time.
+  - `Use Originium` specifies the number of Originium to use at most. It is used one at one time.
```

Comment:
```suggestion
  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Multiple medicines may be used at a time.
  - `Use Originium` specifies the number of Originium to use at most. It is used one at a time.
```

Reply:
Question: Aren't potions also used one at a time?

Reply:
Small potion may not have enough santy for stages. We used `using times` to limit actions. Now we need to set the count of potion.